Climate_police/homework 7/temperature.py

```diff
@@ -0,0 +1,178 @@
+"""temperature.py"""
+
+import wget
+import os
+import zipfile
+
+import urllib3
+import certifi
+import sys
+import glob
+
+import numpy as np
+import pandas as pd
+
+import matplotlib.pyplot as plt
+import plotly.offline as py
+import seaborn as sns
+
+def download_if_needed(URL, filename):
+    """
+    Download from URL to filename unless filename already exists
+
+    """
+
+    if os.path.exists(filename):
+        print('The File already exists !!')
```

Comment:
You can simply use "pass" instead of print the sentence, otherwise, whenever you use functions that use download_if_needed, it will print "The File already exists".